### ISSUE TITLE
Build error when building nexe vm with docker release build

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/release-scripts#9]
```
/Unvanquished/src/sgame/botlib/bot_load.cpp:54:2: error: 
#error
1 error generated.
make[6]: *** [CMakeFiles/sgame-nacl.dir/build.make:684: CMakeFiles/sgame-nacl.dir/src/sgame/botlib/bot_load.cpp.obj] Error 1
make[5]: *** [CMakeFiles/Makefile2:437: CMakeFiles/sgame-nacl.dir/all] Error 2
make[5]: *** Waiting for unfinished jobs....
make[4]: *** [Makefile:84: all] Error 2
make[3]: *** [CMakeFiles/nacl-vms.dir/build.make:112: nacl-vms-prefix/src/nacl-vms-stamp/nacl-vms-build] Error 2
make[2]: *** [CMakeFiles/Makefile2:110: CMakeFiles/nacl-vms.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:122: CMakeFiles/nacl-vms.dir/rule] Error 2
make: *** [Makefile:131: nacl-vms] Error 2
ERROR INTERNAL: vm build failed
The command '/bin/sh -c /build-release -j`nproc` ${targets}' returned a non-zero code: 3
```

Note: I didn't tried to build nexe since a long time, even outside of docker, some maybe only the shared lib builds?

**Comment by illwieckz** [comment]
This is the way it is configured on Linux (without docker) with the `build-release` script:

```
cmake -H/home/illwieckz/dev/Unvanquished -B/home/illwieckz/dev/Unvanquished/build/target/vm '-GUnix Makefiles' -DBUILD_GAME_NATIVE_DLL=OFF -DBUILD_GAME_NATIVE_EXE=OFF -DCMAKE_BUILD_TYPE=RelWithDebInfo -DUSE_BREAKPAD=ON -DBUILD_GAME_NACL=ON -DBUILD_GAME_NACL_NEXE=ON -DBUILD_CGAME=ON -DBUILD_SGAME=ON -DBUILD_CLIENT=OFF -DBUILD_TTY_CLIENT=OFF -DBUILD_SERVER=OFF
```

And the way it is configured by my Makefile (same host):

```
cmake '/home/illwieckz/dev/buildme-unvanquished/Unvanquished' -B'/home/illwieckz/dev/buildme-unvanquished/build/vms/test' \
	-D'USE_BREAKPAD=ON' -D'CMAKE_BUILD_TYPE=RelWithDebInfo' -D'USE_LTO'='OFF' \
	-D'BUILD_GAME_NACL=ON' -D'BUILD_GAME_NACL_NEXE=ON' -D'BUILD_GAME_NATIVE_EXE=OFF' -D'BUILD_GAME_NATIVE_DLL=OFF' \
	 \
	-D'EXTERNAL_DEPS_DIR=/home/illwieckz/dev/buildme-unvanquished/build/deps' \
	-D'BUILD_SERVER=OFF' -D'BUILD_CLIENT=OFF' -D'BUILD_TTY_CLIENT=OFF' \
	-D'BUILD_SGAME=ON' -D'BUILD_CGAME=ON' \
	-D'DAEMON_DIR=/home/illwieckz/dev/buildme-unvanquished/Daemon' \
	-G'Unix Makefiles'
```

The `build-release` script fail but my Makefile works, I don't know why.

**Comment by slipher** [comment]
https://github.com/DaemonEngine/Daemon/pull/443

**Comment by illwieckz** [comment]
Fixed by DaemonEngine/Daemon#443
